packages/block-library/src/post-breadcrumbs/style.scss

```
@@ -0,0 +1,27 @@
+.wp-block-post-breadcrumbs {
```

```suggestion
.wp-block-post-breadcrumbs {
	// This block has customizable padding, border-box makes that more predictable.
	box-sizing: border-box;
```

See: https://github.com/WordPress/gutenberg/issues/43465